### PR TITLE
Remove pairs from EVM chain tokelists.json

### LINKED ISSUE #17108: Add a command for adding new tokens to tokenlists
**Issue**
We need to be able to add new token with make-command.

Currently, we have some make-commands like check, fix, update-auto and manual-auto.
Add a command for adding new token to the tokenlist.json.

Example: 
`make tokenlist add c60_t0x4Fabb145d64652a948d72533023f6E7A623C7C53`

**Note:**
As we have different chains (smartchain, polygon, binance, ethereum) with token lists, may be we should parse a token prefix to get appropriate chain (example: `c60_t0x4Fabb145d64652a948d72533023f6E7A623C7C53` - > `c60` -> "ethereum" -> path to tokenlist is `blockchains/ethereum/tokenlist.json`).

### LINKED ISSUE #17642: Tokenlist changes
- Remove all pairs from EVM chain tokenlists (except of binance chain)
- Add tokenlist-extended.json
- Add command for adding token to tokenlist-extended.json

### PR DESCRIPTION
**Pull request by unanoc**
Closes #17642
Closes #17108

### CODE REVIEW

File: blockchains/polygon/tokenlist.json [hunk 1]
```diff
@@ -11,29 +11,7 @@
...
-                    "base": "c966_t0x8f3Cf7ad23Cd3CaDbD9735AFf958023239c6A063"
-                },
-                {
-                    "base": "c966_t0xc2132D05D31c914a87C6611C10748AEb04B58e8F"
-                },
-                {
-                    "base": "c966_t0xD6DF932A45C0f255f85145f286eA0b292B21C90B"
-                }
-            ]
+            "pairs": []
```
Comment (vikmeup) [inline blockchains/polygon/tokenlist.json:14]: we need to remove `"pairs": []` from json for EVM

File: internal/config/config.go [hunk 2]
```diff
@@ -28,6 +29,7 @@ type (
 
 	URLs struct {
 		TWAssetsApp string `mapstructure:"tw_assets_app"`
```
Comment (vikmeup) [inline internal/config/config.go:31]:
```suggestion
		AssetsApp string `mapstructure:"tw_assets_app"`
```

same below

File: README.md [hunk 3]
```diff
@@ -58,6 +58,7 @@ There are several scripts available for maintainers:
 - `make fix` -- Perform automatic fixes where possible
 - `make update-auto` -- Run automatic updates from external sources, executed regularly (GitHub action)
 - `make add-token token=c60_t0x4Fabb145d64652a948d72533023f6E7A623C7C53` -- Create `info.json` file as asset template.
+- `make add-tokenlist-extended token=c60_t0x4Fabb145d64652a948d72533023f6E7A623C7C53` -- Adds a token to tokenlist-extended.json.
```
Comment (vikmeup) [inline README.md:61]: I think we need to support both command here:
`make add tokenlist token=c60_t0x4Fabb145d64652a948d72533023f6E7A623C7C53`
`make add tokenlist-extended token=c60_t0x4Fabb145d64652a948d72533023f6E7A623C7C53`

File: internal/config/config.go [hunk 4]
```diff
@@ -28,8 +28,8 @@ type (
 	}
 
 	URLs struct {
-		TWAssetsApp string `mapstructure:"tw_assets_app"`
-		TWLogo      string `mapstructure:"tw_logo"`
+		AssetsApp string `mapstructure:"assets_app"`
+		TWLogo    string `mapstructure:"tw_logo"`
```
Comment (vikmeup) [inline internal/config/config.go:32]:
```suggestion
		Logo    string `mapstructure:"logo"`
```

same here

File: README.md (hunk 3 above)
Comment (leedaniil) [reply README.md:62]: added

File: blockchains/ethereum/tokenlist-extended.json [hunk 5]
```diff
@@ -0,0 +1,21 @@
+{
+    "name": "Trust Wallet: Ethereum",
+    "logoURI": "https://trustwallet.com/assets/images/favicon.png",
+    "timestamp": "2022-01-26T20:58:24.420421",
+    "tokens": [
+        {
```
Comment (vikmeup) [inline blockchains/ethereum/tokenlist-extended.json:6]: ideally add the token that does not exist in the `tokenlist`

File: internal/manager/commands.go [hunk 6]
```diff
@@ -72,6 +75,50 @@ var (
 			}
 		},
 	}
+
+	addTokenlistCmd = &cobra.Command{
```
Comment (vikmeup) [inline internal/manager/commands.go:79]: why duplicate `addTokenlistCmd` and `addTokenlistExtendedCmd` run command? it just a different file, create a function for it

File: internal/manager/helpers.go [hunk 7]
```diff
@@ -67,3 +72,42 @@ func CreateAssetInfoJSONTemplate(token string) error {
...
+	return libFile.CreateJSONFile(tokenListPath, &tokenlist.Model{
+		Name:      fmt.Sprintf("Trust Wallet: %s", coin.Coins[chain.ID].Name),
+		LogoURI:   config.Default.URLs.Logo,
+		Timestamp: time.Now().Format(config.Default.TimeFormat),
+		Tokens:    oldTokenList.Tokens,
+		Version:   tokenlist.Version{Major: oldTokenList.Version.Major + 1},
+	})
+}
+
+func GetChainAndTokenID(token string) (string, coin.Coin, error) {
```
Comment (vikmeup) [inline internal/manager/helpers.go:101]: this function should exist in `go-primitives` already

**Pull request merged**